demisto_sdk/commands/validate/validators/PA_validators/PA132_is_valid_default_datasource.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+from typing import Dict, Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+
+
+class IsValidDefaultDataSourceNameValidator(BaseValidator[ContentTypes]):
+    error_code = "PA132"
+    description = "Validate that the pack_metadata contains a valid default datasource, when provided."
+    rationale = "Wizards and other tools rely on the default datasource to be set."
+    error_message = (
+        "Pack metadata contains an invalid 'defaultDataSource': {0}. "
+        "Please fill in a valid datasource integration, one of these options: {1}."
+    )
+    fix_message = "Set the 'defaultDataSource' for '{0}' pack to the '{1}' integration (changed from display name to id)."
```

Comment:
Updated the 'defaultDataSource' for the '{0}' pack to use the '{1}' integration ID instead of the display name that previously used.